app/middleware/services.js

```diff
@@ -149,13 +149,15 @@ function getDisplayValuesMapper(location) {
 function prepareForRender(req, res, next) {
   const open = req.query.open || false;
   const location = res.locals.location;
+  const labelOpenOnly = 'Open now';
```

Comment:
I'd be inclined to move the labels into the view as variables rather than have them here. There isn't any logic that needs testing (one of the reasons for having stuff in the middleware).

https://mozilla.github.io/nunjucks/templating.html#set